runner/runner.go

```diff
@@ -231,59 +240,85 @@ func (r *Runner) run(ctx context.Context, checkMap map[string]*Check, list bool,
 		}
 	}
 
+	errs := make(chan responseError, len(currentCheckList))
+	responses := make(chan *Response, len(currentCheckList))
```

Comment:
As `responseError` contains a `*Response`, could we get rid of the `responses` channel and rename `errs` to `results` or something similar? Then we have only one channel. (I believe you suggested this a while ago.)